.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.17.5
```

Comment:
What's this file for?

Reply:
ah, I use asdf to manage software versions, will remove it

Reply:
Ah, gotcha. I hadn't heard of that one before. Thanks